### ISSUE TITLE
Changing the Warp Effects in Warp Theory.

### ISSUE THREAD

**Issue by Dream-Master** [GTNewHorizons/NewHorizons#272]
Changing the Warp Effects in Warp Theory (Get lower when not touch Thaumcraft) to be working the same way like it works in Thaumcraft.

**Comment by Namikon** [comment]
As I have a little bit of free time, I tend to browse a bit of sourcecode now and then.
I already changed a few things in warp theory: First, the Arcane Litmus paper now behaves properly, and will display in normal human words how many warp one has. Second, the way how the warp purging bauble works: It selects one warp effect that is appropriate for your level, triggers it, and removes the amount of warp that is assigned to that effect. For the wither, this would be 4 permanent warp. This continues until no warp effect can be found, because you don't have enough warp for any effect. (Which will eventually end up for you with 5 warp in total) For me, since i had ~130 warp, that meant about 7 wither spawns. (You can take a look at my old base what else happend there. It is quite terrifying)

In other words:
Even if you have taken all the precautions, made (like me) half a stack of soap and bath-salts in advance, **the wither will spawn**, and it doesn't matter if you are under warp-ward or not. Even purging warp will cause all effects, and that quite often. So either: Deal with it, and build yourself an area in which the wither can spawn safely (Wand focus warding), or, keep your warp level below / around 50, which is enough to trigger the required tab in the book.

Not 100% sure about the effects in general however. They seem to appear quite often, I will check that soon

**Comment by Dream-Master** [comment]
I will think about new Warp effects we can add ;-)

**Comment by Namikon** [comment]
Can we close this?

**Comment by Dream-Master** [comment]
Not sure because you said before you wsnt to add or change things. I think about the warp effects that still not go a way like in thaumcraft.

**Comment by Namikon** [comment]
Ah that, no, I cleared that. It thought it doesn't follow the rules, but it does. I reversed the mods code, and everything makes sense. I explained that in the post from 5.8.. It was about the amulet, but it's also valid for all effects. It happend that I also used profane wands back then, and that's why I always had a little gain in warp, which, of course, gave me warp effects and reset the timer. So, except if you want to add more effects, there is nothing wrong with this mod

**Comment by Dream-Master** [comment]
So than we can close this.
